.github/workflows/end2end-test.yml

```diff
@@ -2,12 +2,12 @@ name: End-to-End Tests
 
 on:
   pull_request:
-    paths:
-    - '!**/*.md'
+    paths-ignore:
+    - '**.md'
```

Comment:
I think all of these list items need to be indented one step further. That's how it looks in the [GitHub docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration), anyway. Not sure if it makes a syntactical difference.